game/static/game/css/landing.css

```
@@ -889,6 +889,126 @@ body {
+  position: relative;
+}
+
+.faq-container {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+.faq-subtitle {
+  text-align: center;
+  color: #a0a5b5;
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)